src/services/query-options.js

```diff
@@ -30,6 +30,9 @@ class QueryOptions {
       options.offset = this._offset;
       options.limit = this._limit;
     }
+    if (this._requestedFields.size) {
```

Comment:
this condition is not correct. As of right now smart field depends on the retrieval of all the fields. What we could do is restrict the attributes only if no smartFields is requested

Reply:
With the huge amount of regression that this project has been having lately, I'm paranoid, and changed it so that the restriction is only applied when a specific flag is passed from forest-express (this is never used from the frontend)